app/models/concerns/form526_rapid_ready_for_decision_concern.rb

```diff
@@ -49,13 +49,16 @@ def notify_mas_tracking
 
   def notify_mas
     notify_mas_tracking
-    client = MailAutomation::Client.new({
-                                          file_number: birls_id,
-                                          claim_id: submitted_claim_id,
-                                          form526: form
-                                        })
-    response = client.initiate_apcas_processing
-    save_metadata(offramp_reason: "mas_notification[#{response.dig('body', 'packetId')}]")
+
+    if Flipper.enabled?(:rrd_mas_notification)
+      client = MailAutomation::Client.new({
+                                            file_number: birls_id,
+                                            claim_id: submitted_claim_id,
+                                            form526: form
+                                          })
+      response = client.initiate_apcas_processing
+      save_metadata(offramp_reason: "mas_notification[#{response.dig('body', 'packetId')}]")
```

Comment:
This is not an offramp, right?

Reply:
```suggestion
      save_metadata(mas_packetId: response.dig('body', 'packetId'))
```
Do we need to check the presence of `response.dig('body', 'packetId')`?